src/WebJobs.Extensions/WebJobs.Extensions.csproj

```diff
@@ -19,8 +19,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.14" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Host.Storage" Version="3.0.11" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.15-dev637091944077950638" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Host.Storage" Version="4.0.0-dev637091944077950638" />
```

Comment:
Can you remind me (I know we talked about it) -- what does this depend on from Host.Storage?

Reply:
The timer extension lives in this project, and it depends on storage.